### PR TITLE
Some project refinements including configurations

### PR DESCRIPTION
**Pull request by ChuckySRB**

### CODE REVIEW

File: src/configs/config_esm3.json [hunk 1]
```diff
@@ -1,22 +1,26 @@
 {
-"emb_dir": "results/tox_esm3ft2",
-"program_mode": "TRAIN_PREDICT_FROM_STORED",
+  "emb_dir": "./emb_dir/tox_esm3ft2",
+  "program_mode": "TRAIN_PREDICT_FROM_STORED",
```
Comment (vladimirkovacevic) [inline src/configs/config_esm3.json:3]: Use RUN_ALL because user probably don't already have embeddings

File: src/configs/config_esm3.json [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
-"wandb_key": "66bdf7f04d7842bb591556f5263dd9c779ca1ce7" 
+  "classifier_type": "MLP",
+  "batch_size": 64,
+  "num_epochs": 3,
+  "lr": 1e-3,
+  "hidden_layers": null,
+
+  "model_folder": "classifier_results/weights",
+  "model_basename": "esm3_model_test",
+  "finetune_model_path": "ESM3_finetuned_epoch_19.pt",
```
Comment (vladimirkovacevic) [inline src/configs/config_esm3.json:20]: Remove. User probably don't have this model

File: src/configs/config_protein_trans.json [hunk 3]
```diff
@@ -6,13 +6,14 @@
     "train": "/home/share/huadjyin/home/chenjunhong/META_AI/datasets/AMP/AMP_2024_08_09_train.tsv",
     "valid": "/home/share/huadjyin/home/chenjunhong/META_AI/datasets/AMP/AMP_2024_08_09_validation.tsv",
     "test": "/home/share/huadjyin/home/chenjunhong/META_AI/datasets/AMP/AMP_2024_08_09_test.tsv",
-    "emb_dir": "/home/share/huadjyin/home/vladimir/META_AI/datasets/AMP/all_embeddings_ptrans",
+    "emb_dir": "./datasets/AMP/all_embeddings_ptrans",
 
-    "emb_type": "PTRANS",
+    "model_type": "PTRANS",
     "model_name_or_path": "Rostlab/prot_t5_xl_uniref50",
     "finetune_model_path": "/home/share/huadjyin/home/vladimir/Metagenome-AI/ptrans_encoder_ft_6.pt",
```
Comment (vladimirkovacevic) [inline src/configs/config_protein_trans.json:13]: Remove

File: src/configs/config_sample.json [hunk 4]
```diff
@@ -1,13 +1,14 @@
 {
     "model_type":"ESM",
     "program_mode": "RUN_ALL",
-    "train": "data/sample_train.tsv",
-    "valid": "data/sample_validation.tsv",
-    "test": "data/sample_test.tsv",
+    "train": "data/sample_train.csv",
```
Comment (vladimirkovacevic) [inline src/configs/config_sample.json:4]: rest of the datasets is TSV, why CSV here?

File: src/configs/config_sample.json [hunk 5]
```diff
@@ -1,13 +1,14 @@
...
+    "valid": "data/sample_validation.csv",
+    "test": "data/sample_test.csv",
     "emb_dir": "emb_dir/sample",
-    "model_folder": "esm_results",
+    "model_folder": "classifier_results/sample",
     "wandb_key": "ba8ab5025007d1a06c3129ba630ed65767461de7",
 
+    "classifier_type": "MLP",
```
Comment (vladimirkovacevic) [inline src/configs/config_sample.json:11]: Maybe use here XGboost?

File: src/configs/config_sample.json (hunk 4 above)
Comment (ChuckySRB) [reply src/configs/config_sample.json]: I changed the data/sample files format from .csv to .tsv to match the rest of the data

**Pull request merged**